xrviz/.travis.yml

```diff
@@ -0,0 +1,28 @@
+sudo: False
+
+language: python
+
+matrix:
+  include:
+    - python: 3.6
+    - python: 3.7
+
+install:
+  # Install conda
+  - wget https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update conda
+
+  # Install dependencies
+  - conda create -n test python=$TRAVIS_PYTHON_VERSION pytest
+  - source activate test
+  - conda install xarray cartopy panel hvplot netcdf4
```

Comment:
I have tested by creating an environment using this command and running the interface. It works fine.